modules/apps/transfer/keeper/relay.go

```diff
@@ -279,14 +279,14 @@ func (k Keeper) OnRecvPacket(ctx sdk.Context, packet channeltypes.Packet, data t
 	if err := k.bankKeeper.MintCoins(
 		ctx, types.ModuleName, sdk.NewCoins(voucher),
 	); err != nil {
-		return err
+		return sdkerrors.Wrapf(err, "failed to mint IBC tokens")
 	}
 
 	// send to receiver
 	if err := k.bankKeeper.SendCoinsFromModuleToAccount(
 		ctx, types.ModuleName, receiver, sdk.NewCoins(voucher),
 	); err != nil {
-		return err
+		return sdkerrors.Wrapf(err, "failed to send coins to receiver %s: %s", receiver.String(), err.Error())
```

Comment:
```suggestion
		return sdkerrors.Wrapf(err, "failed to send coins to receiver %s", receiver.String())
```
Since we wrap the err, we don't need to include it in the error string as well

Reply:
Thank you, I've changed it